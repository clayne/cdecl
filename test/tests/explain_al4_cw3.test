EXPECTED_EXIT=65
cdecl <<END
explain _Alignas(4) char c : 3
END
