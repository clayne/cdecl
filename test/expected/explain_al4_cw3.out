explain _Alignas(4) char c : 3
        ^
9: error: bit fields can not be aligned
